### PR TITLE
feat: add basic quickstart ui using frontend sdk (ENG-372)

### PR DESCRIPTION
**Pull request by taivokasper**
### Task / problem
Task: [Spin up front-end when running a copilot locally](https://linear.app/nftport/issue/ENG-372/spin-up-front-end-when-running-a-copilot-locally)
Allows quickly to see what OpenCopilot is all about. Includes a simple frontend where you can communicate with the copilot.

### Changes made
By default hosts frontend under `/ui` url

### CODE REVIEW

File: opencopilot/app.py [hunk 1]
```diff
@@ -17,6 +24,17 @@
 
 app.include_router(main_router.router, prefix="/v0")
 
+html_template_path = join(dirname(opencopilot.__file__), "html")
+# TODO Taivo & Kristjan: remove once sdk has been released
```
Comment (taivokasper) [inline opencopilot/app.py:28]: Remove once react SDK is released. Waiting behind @kristjanpeterson1

File: opencopilot/html/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>OpenCopilot demo</title>
+    <!-- Include your library -->
+    <script src="js/opencopilot.js"></script>
```
Comment (taivokasper) [inline opencopilot/html/index.html:8]: Replace with external path once react SDK is released. Waiting behind @kristjanpeterson1

**Comment by taivop** [comment]
@kristjanpeterson1 before merging please make sure the UI covers the whole page (width & height) responsively -- or looks otherwise nice.

**Review by KasparPeterson — APPROVED** [review]

**Comment by taivokasper** [comment]
@kristjanpeterson1 these PRs are unfortunately stacked on top of each other so that I could make small PRs and continue my work. There were meant to be merged in the following order:
1. [Weaviate api key](https://github.com/opencopilotdev/opencopilot/pull/74)
2. [UI](https://github.com/opencopilotdev/opencopilot/pull/73)
3. [deployment (needs extra work and cleanup once other PRs are merged)](https://github.com/opencopilotdev/opencopilot/pull/72)

**Pull request merged**